packages/examples/fortune-v2/recording-oracle/src/modules/job/job.service.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Injectable } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+import { EscrowClient, EscrowStatus, NETWORKS, StorageClient } from "@human-protocol/sdk";
+import axios from "axios";
+import { ethers } from "ethers";
+
+import { JobSolutionRequestDto } from "./job.dto";
+
+@Injectable()
+export class JobService {
+  constructor(private configService: ConfigService) {}
+
+  async processJobSolution(jobSolution: JobSolutionRequestDto): Promise<any> {
+    // Initialize Escrow Client
+    const network = NETWORKS[jobSolution.chainId];
+
+    const provider = new ethers.providers.JsonRpcProvider(this.configService.get<string>("ethereum.jsonRpcUrl", ""));
```

Comment:
can we get the config properties in a typed way? if it is not possible, we should use constants to with the string "ethereum.jsonRpcUrl"